src/ray/raylet/placement_group_resource_manager.cc

```diff
@@ -57,6 +57,7 @@ bool NewPlacementGroupResourceManager::PrepareBundle(
 
   auto resource_instances = std::make_shared<TaskResourceInstances>();
   bool allocated =
+      !cluster_resource_scheduler_->GetLocalResourceManager().IsLocalNodeDraining() &&
```

Comment:
Instead of this can we just do 

```
if (cluster_resource_scheduler_->GetLocalResourceManager().IsLocalNodeDraining()) {
    return false
}
```
? I feel like that's easier to understand?